lld/ELF/LinkerScript.cpp

```diff
@@ -131,6 +131,156 @@ uint64_t ExprValue::getSectionOffset() const {
   return getValue() - getSecAddr();
 }
 
+std::function<ExprValue()> ScriptExpr::getExpr() const {
+  switch (kind_) {
+  case ExprKind::Constant:
+    return static_cast<const ConstantExpr *>(this)->getConstantExpr();
+  case ExprKind::Dynamic: {
```

Comment:
Having second thoughts about the name "Dynamic"; that word implies change or motion, which doesn't actually seem to fit here, since the logic doesn't actually change. It's a variable, not a time series. Maybe "Generic"? "Functor"?

Reply:
I changed the name to `FallbackExpr` but we can always change it if we have a better one.